### ISSUE TITLE
[bug] Leaderboard shows incorrect user as claiming an item when it's gifted

### ISSUE THREAD

**Issue by matthewbrandt** [whitep4nth3r/pantherworld#53]
I cannot really explain this any better so I'll just provide a screenshot and a timelog:
- Item spawns
- User matty_twoshoes claims item
- User matty_twoshoes gifts item to matty_nshoes
- Leaderboard now shows latest claim as item by matty_nshoes

![Screenshot 2024-07-30 at 12 15 30](https://github.com/user-attachments/assets/5004a694-38c0-4754-a6ff-c05cac96dc64)

**Comment by whitep4nth3r** [comment]
Yeah the latest "claim" is really just the "most recent person who acquired an item".
